superset/config.py

```diff
@@ -124,6 +124,8 @@ def _try_json_readsha(filepath, length):  # pylint: disable=unused-argument
 # (gunicorn, nginx, apache, ...) timeout setting to be <= to this setting
 SUPERSET_WEBSERVER_TIMEOUT = 60
 
+SUPERSET_DASHBOARD_PERIODICAL_REFRESH_LIMIT = 0
+SUPERSET_DASHBOARD_PERIODICAL_REFRESH_WARNING_MESSAGE = None
```

Comment:
Would you mind adding a comment about what these config params do too?

Reply:
added short description.